components/AudioPlayer.module.css

```
@@ -0,0 +1,236 @@
+.audio_player_section_container {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	/* @@ updated */
+	padding: 0 3rem;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: 10% 30% 45% 15%;
+	justify-content: center;
+	align-items: center;
+	/* @@ updated */
+}
+
+.image_wrapper {
+	/* @@ updated */
+	position: relative;
+	width: 60%;
+	height: 90%;
+	/* @@ updated */
+	margin: 1rem auto;
+	box-shadow: 0 0 20px black;
+}
+
+.episode_info_section {
+	/* @@ updated */
+	height: 100%;
+	/* align-self: flex-start; */
+	align-self: center;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: auto auto;
+	gap: 1rem;
+	padding-bottom: 1.5rem;
+	/* @@ updated */
+}
+
+.episode_title {
+	text-align: center;
+	/* @@ updated */
+	margin: 1rem 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 0.5rem;
+	/* @@ updated */
+}
+
+/* @@ updated */
+.episode_title p {
+	font-weight: 600;
+	margin: 0;
+}
+/* @@ updated */
+
+.episode_details {
+	/* @@ updated */
+	position: relative;
+	padding: 0 1rem;
+	/* @@ updated */
+	overflow: scroll;
+	text-align: start;
+}
+
+.episode_details::-webkit-scrollbar {
+	display: none;
+}
+
+/* @@ updated */
+.episode_details p {
+	margin: 0;
+}
+/* @@ updated */
+
+.audioPlayer {
+	--primary: grey;
+	--secondary: white;
+	/* @@ updated */
+	text-align: center;
+	align-self: flex-start;
+}
+
+.audioPlayerPlaybutton {
+	display: flex;
+	justify-content: center;
+	gap: 20px;
+	/* @@ updated */
+	margin-bottom: 1rem;
+}
+
+.forwardBackward {
```

```suggestion
.controlThirty {
```
**Suggestion**: it was used for the forward button as well. Naming `forwardBackward` might confuse so I suggest that name more general name like this.